midp/background.js

```diff
@@ -54,7 +54,19 @@ function backgroundCheck() {
   fgMidletNumber = (bgServer == 2) ? 1 : 2;
   fgMidletClass = MIDP.manifest["MIDlet-" + fgMidletNumber].split(",")[2];
 
-  DumbPipe.close(DumbPipe.open("backgroundCheck", {}));
+  if (!MIDlet.shouldStartBackgroundService ||
```

Comment:
Instead of checking if the property is truthy, I would add a default function (to config/default.js) that always returns false.